eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -61,10 +70,13 @@ else {
     body                  = $PRBody
     maintainer_can_modify = $true
   }
+  Write-Host "Got Here 6"
 
   $resp = Invoke-RestMethod -Method POST -Headers $headers `
-                            https://api.github.com/repos/$RepoOwner/$RepoName/pulls `
-                            -Body ($data | ConvertTo-Json)
+    https://api.github.com/repos/$RepoOwner/$RepoName/pulls `
+    -Body ($data | ConvertTo-Json)
+  Write-Host "Got Here 7"
   $resp | Write-Verbose
+  Write-Host "Got Here 8"
```

Comment:
I realize these are here for diagnostics purposes but we're definitely going to want to remove them before merging.

Reply:
Fixed